app/controllers/homeworks_controller.rb

```diff
@@ -1,8 +1,17 @@
 class HomeworksController < ApplicationController
   before_action :set_assignment, only: [ :edit, :update, :destroy ]
 
+  @@old_hw = nil
```

Comment:
I use this for save invalid homework. It allows me redirect user to new_homework_path instead "/homeworks". Is it good? Or remove this and return "render "new""?